pug/modules/menu.pug

```diff
@@ -0,0 +1,83 @@
+.header-top
+				nav.navbar.navbar-expand-md.navbar-dark
+					.logo
+						a
+							span M
+							| adala
+					button.navbar-toggler(type='button', data-toggle='collapse', data-target='#collapsibleNavbar')
+						span.navbar-toggler-icon
+					#collapsibleNavbar.collapse.navbar-collapse
+						ul.navbar-nav
+							li.nav-item
+								a.nav-link(href='../pages/home_01.html') TRANG CHỦ
+							li.nav-item
+								a.nav-link(href='../pages/10_about.html') GIỚI THIỆU
+							li.nav-item#dropdown
+								a.nav-link(href='#') SẢN PHẨM
+									//- .dropdown
+									//- 	div.nav-column
+									//- 		h3 DƯỠNG DA
+									//- 		ul
+									//- 			li
+									//- 				a Áo khoác
+									//- 			li
+									//- 				a Bộ áo liền quần
+									//- 			li
+									//- 				a Dresses
+									//- 			li
+									//- 				a Quần váy
+									//- 			li
+									//- 				a Quần short
+									//- 			li
+									//- 				a Quần jean
+									//- 				li
+									//- 				a Đồ đan
+									//- 				li
+									//- 				a Áo nỉ
+									//- 	div.nav-column
+									//- 		h3 NƯỚC HOA
+									//- 		ul
+									//- 			li
+									//- 				a Áo cánh/ Áo sơ mi
+									//- 			li
+									//- 				a Túi xách và Balo
+									//- 			li
+									//- 				a Khăn quàng cổ và Foulards
+									//- 			li
+									//- 				a Dệt kim Hats và Glovers
+									//- 			li
+									//- 				a Vớ và Socks
+									//- 			li
+									//- 				a Jewerly
+									//- 				li
+									//- 				a Bị bảo vệ
+									//- 				li
+									//- 				a Giày
+									//- 	div.nav-column
+									//- 		h3 TRANG SỨC
+									//- 		ul
+									//- 			i
+									//- 				a Áo cánh/ Áo sơ mi
+									//- 			li
+									//- 				a Túi xách và Balo
+									//- 			li
+									//- 				a Khăn quàng cổ và Foulards
+									//- 			li
+									//- 				a Dệt kim Hats và Glovers
+									//- 			li
+									//- 				a Vớ và Socks
+									//- 			li
+									//- 				a Jewerly
+									//- 			li
+									//- 				a Bị bảo vệ
+									//- 			li
```

Comment:
code không dùng xóa đi 
dùng vòng lặp để build luôn li cho anh

Reply:
ví du: 
var breadcrumb = [
        { key: "Account Settings", value: "#"},
        { key: "Other Information", value: "#"},
        { key: "General Information", value: "#"},
        { key: "Change Password", value: "#"}
      ];
each val, index in breadcrumb
            - var item = index === 0
            li.nav-item
              a(
                data-toggle='tab'
                href= `#${val.key}`
                class= item ? 'nav-link active' : 'nav-link'
              )= val.value